src/compounds/product-table/src/generics.ts

```diff
@@ -76,6 +78,16 @@ const formatters: { [unit: string]: (value: number) => string } = {
     if (megabytes >= 1000) return `${Math.floor(megabytes / 1000)}GB`
 
     return `${megabytes || 0}MB`
+  },
+
+  'exchange-rate': (value, count = 4) => {
+    const regex = new RegExp(`^-?\\d+(?:\\.\\d{0,${count}})?`)
+    const matches = value.toString().match(regex)
+    const withDecimals = matches?.length ? matches[0] : ''
+    const [before = '', after = ''] = withDecimals.split('.')
+    const val = `${before}.${after.padEnd(count, '0')}`
+
+    return val
```

Comment:
could we instead do:

```javascript
return Intl.NumberFormat('en-GB' , {maximumFractionDigits: count, minimumFractionDigits: count}).format(value)
```

Reply:
`Intl.NumberFormat` rounds up when the number has more than 4 decimal places but Akanksha wanted to get the first 4 decimal places without rounding it up.